models/trove/arbitrum/trove_arbitrum_trades.sql

```diff
@@ -63,12 +79,14 @@ select
     project_contract_address,
     cast(null as varchar(5)) as aggregator_name,
     cast(null as varchar(5)) as aggregator_address,
-    tx_hash,
+    mp.tx_hash,
     mp.block_number,
     tx.from as tx_from,
```

Comment:
Hey @Hosuke! `from` is a reserved word on DuneSQL so it needs to be escaped when it's used for selecting a column. Changing this to be tx.`from` should do the trick 🤞

Reply:
like this?
```suggestion
     tx.`from` as tx_from,
```

Reply:
Exactly 👍